src/v2/controllers/FormController.js

```diff
@@ -107,12 +106,17 @@ export default Controller.extend({
           // the SIW can proceed to the next step without showing error
           this.updateAppStateWithNewIdx(error);
         } else {
-          model.trigger('error', model, {'responseJSON': error}, true);
-          this.toggleFormButtonState(false);
+          this.showFormErrors(model, error);
         }
       });
   },
 
+  showFormErrors (model, error) {
+    const convertedErrors = IonResponseHelper.convertFormErrors(error);
+    model.trigger('error', model, convertedErrors, convertedErrors.responseJSON.errorCauses.length ? false : true);
```

Comment:
when `errorCauses` is not empty, shall we still display top lever errorSummary ?

Reply:
courage baseForm either show validationErrors or errorSummary. So if errorCauses is not empty then we don't show top level errors.